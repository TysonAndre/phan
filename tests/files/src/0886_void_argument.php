<?php
echo json_encode(debug_print_backtrace());
