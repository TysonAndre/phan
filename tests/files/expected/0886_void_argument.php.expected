%s:2 PhanTypeVoidArgument Cannot use void return value debug_print_backtrace() as a function argument
